requirements.txt

```
@@ -2,7 +2,7 @@
 Pygments >= 2.9.0
 # Sphinx 6.1.0 broke copying images in parallel builds; fixed in 6.1.2
 # See https://github.com/sphinx-doc/sphinx/pull/11100
-Sphinx >= 5.1.1, != 6.1.0, != 6.1.1
+Sphinx >= 5.1.1, != 6.1.0, != 6.1.1, !=7.2.0
```

Testing new release:
```suggestion
Sphinx >= 5.1.1, != 6.1.0, != 6.1.1
```